semgrep/semgrep/rule_schema.yaml

```diff
@@ -443,47 +446,50 @@ properties:
             - message
             - languages
             - severity
-          oneOf:
+          anyOf:
```

Comment:
I worry about this extra complexity here. Can we still require a pattern?

Reply:
We don't want to require a pattern, so that semgrep can filter just based on the precense of a dep at a specific version. It might be better to have a `mode: dependencies`?

Reply:
Resolved by moving under `patterns`